visu3d/dc_arrays/camera_spec.py

```diff
@@ -118,14 +119,60 @@ def w(self) -> int:
 
   @property
   def hw(self) -> tuple[int, int]:
-    """`(Height, Width)` in pixel (for usage in `(i, j)` coordinates)."""
+    """`(Height, Width)` in pixel (for usage in `(u, v)` coordinates)."""
     return (self.h, self.w)
 
   @property
   def wh(self) -> tuple[int, int]:
-    """`(Width, Height)` in pixel (for usage in `(u, v)` coordinates)."""
+    """`(Width, Height)` in pixel (for usage in `(i, j)` coordinates)."""
     return (self.w, self.h)
 
+  @property
+  def fw(self) -> float:
```

Comment:
Contrary to the resolution which is always constant across all camera stacked together, focal length can vary for individual cameras.

This means we should wrap the property around `@vectorize_method` similarly to the scale property in https://github.com/google-research/visu3d/blob/11c8f970336cd2440a355edd35e980c9f9813524/visu3d/dc_arrays/transformation.py#L224

This would allow for example to vary the focal length across camera, like:

```python
cams = v3d.PinholeCamera(
    resolution=(h, w),
    focal_in_px=[200, 240, 260, 300],
)
cams.shape == (4,)  # 4 cameras stacked together with focal 200, 240,...
```

This also mean `focal_length` should likely be a `FloatArray['*shape 2']`